app/routes/events/components/Admin.js

```diff
@@ -93,7 +93,7 @@ const Admin = ({ actionGrant, event, deleteEvent }: Props) => {
           <li>
             <AnnouncementInLine
               placeholder="Skriv en kunngjøring til alle påmeldte..."
```

Comment:
Is this placeholder used for anything now?

Reply:
nope